Makefile

```diff
@@ -1,11 +1,11 @@
-PYTEST_CMD=TESTING=true pytest tests -n 4 -vv
+PYTEST_CMD=TESTING=true poetry run pytest test -n 4 -vv
 
 SHELL := /bin/bash # Use bash syntax
 
 # dev aliases format and lint
-RUFF=ruff app tests
-BLACK=black app tests
-MYPY=mypy app tests
+RUFF=poetry run python -m ruff app test
+BLACK=poetry run python -m black app test
+MYPY=poetry run python -m mypy app test
```

Comment:
poetry run was not enough ? why "poetry run python -m" ?

Reply:
I fixed it